hs_core/management/commands/name_norm_test.py

```diff
@@ -0,0 +1,558 @@
+# -*- coding: utf-8 -*-
+"""
+Test name canonicalization
+
+"""
+
+from django.core.management.base import BaseCommand
+from hs_core.search_indexes import normalize_name
+
```

Comment:
The names you gave me are somewhat indicative of both kinds of names. I can run tests on existing names as well. The normalizations do not remove other annoying variances: first initial rather than first name, leave out middle initial, etc.